package.json

```diff
@@ -34,21 +35,49 @@
     "merge-stream": "^1.0.1",
     "path": ">=0.12.7",
     "semver": "^6.0.0",
-    "stylelint": "^11.1.1",
-    "stylelint-config-wordpress": "^15.0.0",
     "underscore": "^1.9.1"
   },
   "devDependencies": {
-    "nodegit": "^0.24.1"
+    "nodegit": "^0.26.5"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint:css": "wp-scripts lint-style",
+    "lint:js": "wp-scripts lint-js",
+    "format:js": "wp-scripts format-js"
+  },
+  "browserslist": [
+    "last 5 versions",
+    "> 1%",
+    "maintained node versions",
+    "not dead"
+  ],
+  "eslintConfig": {
+    "extends": [
+      "plugin:@wordpress/eslint-plugin/es5"
+    ],
+    "env": {
+      "browser": true,
+      "commonjs": true,
+      "node": false
+    }
+  },
+  "stylelint": {
+    "extends": [
+      "stylelint-config-wordpress/scss"
```

Comment:
Does this still work with the new dependencies?

Reply:
Yeah. Same thing.